src/index.ts

```diff
@@ -93,14 +92,15 @@ export = (app: Application) => {
     })
   }
 
-  async function getAssociatedPullRequests (github: GitHubAPI, { owner, repo, branchName }: { owner: String, repo: string, branchName: string }): Promise<{ owner: string, repo: string, number: number }[]> {
+  async function getAssociatedPullRequests (github: GitHubAPI, { owner, repo, commitId }: { owner: String, repo: string, commitId: string }): Promise<{ owner: string, repo: string, number: number }[]> {
```

Comment:
```suggestion
  async function getAssociatedPullRequests (github: GitHubAPI, { owner, repo, headRefOid }: { owner: String, repo: string, headRefOid: string }): Promise<{ owner: string, repo: string, number: number }[]> {
```
`commitId` isn't a term that is used in GitHub's API. I'd rather go for an existing one like `sha` or `headRefOid` or something like that.